tests/functional/demo/orion_config.yaml

```diff
@@ -6,6 +6,7 @@ max_trials: 100
 algorithms:
   gradient_descent:
     learning_rate: 0.1
+    # dx_tolerance: 1e-7
```

Comment:
Why comment out? Should we remove it?

Reply:
To show that it is the default, but not used here

Reply:
ok, that's fine